conf/config.py

```diff
@@ -200,6 +200,8 @@ class BaseConfiguration(object):
     # repositories that should be searched for unpublished images, specifically because of EUS base images
     UNPUBLISHED_EXCEPTIONS: list[dict[str, str]] = []
 
+    IMAGE_REPOSITORIES_REMOTE_URL = None
```

Comment:
```suggestion
    COMPLIANCE_PRIORITY_REPOSITORIES_REMOTE_FILE = None
```
"image_repositories" is somewhat broad, would it be more suitable to use a more specific name like "compliance_priority_repositories" to clearly indicate its purpose for compliance priority cases?